lib/matplotlib/tests/tinypages/some_plots.rst

```diff
@@ -174,3 +174,18 @@ Plot 21 is generated via an include directive:
 Plot 22 uses a different specific function in a file with plot commands:
 
 .. plot:: range6.py range10
+
+Plots 23 through 25 use output-base-name.
```

Comment:
Why do we need 3? / What do we get more compared to one plote?

Reply:
The second one probably isn't needed (it was when I had the counter thing but I removed that). The third one is testing the case when the plot comes from a file, which is a completely separate code path in the extension.